src/lib/state/actions/deposit.js

```diff
@@ -14,7 +14,7 @@ import {
 export const publish = (record, formik) => {
   return async (dispatch, getState, config) => {
     const controller = config.controller;
-    controller.publishDraft(record, {
+    await controller.publishDraft(record, {
```

Comment:
minor: `return` would have worked here too instead actually.It's clearer to me with `return` but that might just be me.

Reply:
@fenekku you are right! It could work too but we said with @ntarocco that it might be better to be explicit with `async...await` and state clearly that the action is synchronous. Also, the `await` indicates that the action implicates a REST API call!

Reply:
@fenekku before changing this, we have aligned on the chat. Did you change your mind? :)
I am ok with any solution, we should align and use the same way everywhere.